packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -1,6 +1,10 @@
 .floating-rate {
     display: flex;
     flex-direction: column;
+    margin-top: 2.1rem;
+    @include mobile {
+        margin-bottom: 6.5rem !important;
```

Comment:
is it possible to not use `!important` here?

Reply:
I've tried without using `!important` but it doesn't affect the `margin-bottom` in the web since there's a default value which will overwrite the margin-bottom `rem`. Will have to use this `!important` to overwrite the dafault value 🙏